src/Common/src/System/Net/Http/HttpHandlerDefaults.cs

```diff
@@ -16,5 +16,6 @@ internal static class HttpHandlerDefaults
         public const bool DefaultPreAuthenticate = false;
         public const ClientCertificateOption DefaultClientCertificateOption = ClientCertificateOption.Manual;
         public const bool DefaultUseProxy = true;
+        public readonly static TimeSpan DefaultConnectTimeout = TimeSpan.FromSeconds(60);
```

Comment:
>There are 164 occurrences of readonly static across the src tree, and 2343 occurrences of static readonly, so looks like the latter. I'll fix this to match.

We discussed this months ago and @weshaggard updated the coding-guidelines with the answer to this.